### DISCUSSION TITLE
[A-L] (2023/24) Foglio 2 - Esercizio 7

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#148] [Esercizi - Bertini]
![Es 7](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/955a36de-db16-4d92-b951-8db2fa3280b6)

**Comment by daromacarlo** [comment]
![Immagine 2023-10-15 161804](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/a265a2ca-5217-4825-af8c-fd3e95d75927)
![Immagine 2023-10-15 161746](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/a31f7ce2-e4ad-49e4-8d79-e6fe3284e689)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/61793029-d838-4405-a19e-61b22bf0cc1d)
@CarloDaRomadev credo tu debba switchare i (2 scelgo 1) con (2 scelgo 2) immagino sia un'errore di distrazione dato che su uno l'hai fatto

**Comment by daromacarlo** [comment]
@CiottoloMaggico si

**Comment by daromacarlo** [comment]
2)
![Immagine 2023-10-16 081124](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/addc4f3a-b41b-470c-8680-36434a1da0fd)

**Comment by Elia-Belli** [comment]
- Modi di formare i 3 gruppi: $\binom{200}{100} \cdot \binom{100}{50} \cdot \binom{50}{50}$
 Il gruppo da 100 è obbligato ad andare nell'aula III, mentre gli altri la I o la II: $1 \cdot 2 = 2$ modi
 Modi di formare i 3 gruppi e dividerli nelle aule: $|\Omega| = \binom{200}{100} \cdot \binom{100}{50} \cdot 2$

- $A = \\{Alyona\ e\ Bogdana\ stessa\ classe\ ma\ non\ Vadik\\}$
Fisso le due amiche in un gruppo e metto vadik in un'altro: $3 \cdot 2 = 6$ modi
_...coming soon..._

**Comment by LeoCosta02** [comment]
@Elia-Belli 
“ Modi di formare i 3 gruppi e dividerli nelle aule: |Ω|=(200100)⋅(10050)⋅2”
La cardinalità di |Ω| non dovrebbe essere solo (200100)⋅(10050),come hanno fatto @CiottoloMaggico e @CarloDaRomadev ?

**Comment by Elia-Belli** [comment]
@LeoCosta02 il mio $\Omega$ e il loro sono differenti per questo differiscono. Loro hanno calcolato solamente le 2 richieste del punto 1), io anche la loro unione dato che c'ero (non richiesta, è extra)

**Comment by LeoCosta02** [comment]
@Elia-Belli va bene,grazie

**Comment by fraxb04** [comment]
allego una soluzione molto simile alle vostre per il punto 2
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/122573871/32a87689-6ebd-42dc-9c08-fe42a75b37b3)

**Comment by Traster-uni** [comment]
[
![soluz_es7_foglio3](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/84441861/81b44ded-6290-4189-a334-517988ff68b1)
](url)
